File: airport/src/main/java/com/example/airport/model/dto/TicketDto.java
package com.example.airport.model.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDate;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class TicketDto {
    private Long id;
    private Long flightNumber;
    private Long clientId;
    private LocalDate departureDate;
    private Integer price;
}

File: MainProject/MainProject/DTO/TicketDto.cs
using System;

namespace MainProject.DTO
{
    public class TicketDto
    {
        public long Id { get; set; }

        public long FlightNumber { get; set; }

        public long ClientId { get; set; }

        public DateTime DepartureDate { get; set; }

        public int Price { get; set; }
    }
}
